docs/develop/basics.md

```diff
@@ -0,0 +1,117 @@
+
+Example of how to construct a `CredentialProvider` from an environment variable:
+
+<SdkExampleTabs snippetId={'API_CredentialProviderFromEnvVar'} />
+
+If you are storing your credentials in a secrets manager such as AWS Secret Manager or a config file, you will first retrieve the credentials from there and then instantiate a `CredentialProvider` from a string:
+
+<SdkExampleTabs snippetId={'API_CredentialProviderFromString'} />
+
+For an example of how to retrieve credentials from AWS Secrets Manager, see our page on [Retrieving a Momento auth token from AWS Secrets Manager](/develop/integrations/aws-secrets-manager).
+
+For general information on creating and refreshing Momento auth tokens, see our page on [Momento authentication with expiring tokens](/develop/guides/working-with-momento-auth-tokens).
+
+## Configuration
+
+You will also need to create a `Configuration` object to configure your Momento Cache client. `Configuration` objects control things such as timeouts, retries, logging, middleware, and more.
+
+Each SDK comes with some pre-built `Configuration` objects to help you get up and running as quickly as possible. We want to do the hard work of
+tuning for different environments for you, so that you can focus on the things that are unique to your business. (We even have a blog series about it! [Shockingly simple: Cache clients that do the hard work for you](https://www.gomomento.com/blog/shockingly-simple-cache-clients-that-do-the-hard-work-for-you))
+
+Each pre-built configurations come with a `latest()` version, which will always give you the latest recommended configuration for a given environment. This version may change slightly from one release of the SDK to the next as we continue to test, tune, and discover better default settings. If you want to ensure that your configuration does not change with an SDK upgrade, we provide fixed versions such as `v1()` which are guaranteed to remain stable from one release to the next. 
+
+You can find the pre-built configurations in our `Configurations` namespace in most SDKs. Here are some of the pre-built configurations that
+you might be interested in.
+
+### Laptop
+
+`Configurations.Laptop` is a development environment, just for poking around on your... laptop :) It has relaxed timeouts and assumes that your network latencies might be a bit high.
+
+<SdkExampleTabs snippetId={'API_ConfigurationLaptop'} />
+
+### Browser
+
+`Configurations.Browser` is a special configuration available in the [web SDK](/develop/sdks/web). It has relaxed timeout settings since latencies can be extremely variable in individual users' browsers, depending on their network.
```

Comment:
```suggestion
`Configurations.Browser` is a configuration in our [web SDK](/develop/sdks/web). It has relaxed timeout settings since latencies can be highly variable in individual users' browsers, depending on their network.
```